images/stackstorm/Dockerfile

```diff
@@ -128,6 +132,13 @@ COPY bin/entrypoint.sh /st2-docker/bin/entrypoint.sh
 COPY bin/st2.sh /st2-docker/bin/st2.sh
 COPY config/local.conf /etc/init/local.conf
 
+# Install inspec
+RUN curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
+
+COPY ./test /st2-docker/test
+COPY bin/test.sh /st2-docker/bin
+RUN /st2-docker/bin/test.sh
```

Comment:
Looking at the Docker build stage in CircleCI, it would be better to run the `tests` step as a last thing in a Dockerfile.